src/inspector_agent.cc

```diff
@@ -567,12 +578,30 @@ void AgentImpl::WaitForDisconnect() {
                            v8::ReadOnly).FromJust();                          \
   } while (0)
 
+void AddCommandLineAPIMethod(const v8::FunctionCallbackInfo<v8::Value>& info) {
```

Comment:
That's why I have a mild preference for `inline`.  It does the same thing as `static` but there is never any ambiguity as to whether it gets applied to a function or a variable declaration because it's only valid for the former.  Helpful when grepping around.